web_src/css/base.css

```diff
@@ -21,6 +21,7 @@
   --border-radius-circle: 50%;
   --opacity-disabled: 0.55;
   --height-loading: 16rem;
+  --min-height-textarea: 132px;
```

Comment:
`calc(1.57142em + 6lh + 2px)` might be an alternative (padding + 6 lines + border), but `lh` unit is not so well supported: https://caniuse.com/?search=lh%20unit.